src/bin/delete-version.rs

```diff
@@ -8,28 +8,29 @@
 #![deny(warnings)]
 
 extern crate cargo_registry;
-extern crate postgres;
+extern crate diesel;
 extern crate time;
 extern crate semver;
 
+use diesel::prelude::*;
+use diesel::pg::PgConnection;
 use std::env;
 use std::io;
 use std::io::prelude::*;
 
 use cargo_registry::{Crate, Version};
+use cargo_registry::schema::versions;
 
 #[allow(dead_code)]
 fn main() {
-    let conn = cargo_registry::db::connect_now_old();
-    {
-        let tx = conn.transaction().unwrap();
-        delete(&tx);
-        tx.set_commit();
-        tx.finish().unwrap();
-    }
+    let conn = cargo_registry::db::connect_now().unwrap();
+    conn.transaction::<_, diesel::result::Error, _>(|| {
+        delete(&conn);
+        Ok(())
+    }).unwrap()
 }
 
-fn delete(tx: &postgres::transaction::Transaction) {
+fn delete(conn: &PgConnection) {
```

Comment:
I think for the scripts in src/bin, adding documentation more like a man page using docopt or clap would be more appropriate and useful than doc comments. WDYT?